gems/pending/util/miq-xml.rb

```diff
@@ -5,43 +5,34 @@
 require 'util/xml/xml_diff'
 require 'util/xml/xml_patch'
 
-class MiqXml  
-	#MIQ_XML_VERSION = 1.0
-	#MIQ_XML_VERSION = 1.1	# Added create_time to root in seconds for easier time conversions
-	MIQ_XML_VERSION = 2.0	# Changed sub-xmls, added namespaces
+class MiqXml

```

Comment:
A separate commit is probably better anyway; gives git a better chance of finding & following the rename later. Otherwise it'll just see it as a removal of one file, and an addition of a very-different-looking file.